cmd/master/main.go

```diff
@@ -1,10 +1,17 @@
 package main
 
-import "fmt"
-import "github.com/aaronang/cong-the-ripper/master"
+import (
+	"flag"
+	"fmt"
+
+	"github.com/aaronang/cong-the-ripper/master"
+)
 
 func main() {
 	fmt.Println("master starting...")
-	m := master.Init()
+	portPtr := flag.Int("port", 8080, "Web server port")
```

Comment:
I am normally really against variable names that contain the type, but it is up to you.